cfme/tests/intelligence/reports/test_canned_corresponds.py

```diff
@@ -93,3 +94,97 @@ def test_cluster_relationships(soft_assert, setup_a_provider):
                     break
         else:
             soft_assert(False, "Hostname {} not found in {}".format(host_name, provider_name))
+
+
+@pytest.mark.tier(3)
+@test_requirements.report
+def test_operations_vm_on(soft_assert, setup_a_provider):
+
+    appliance = get_or_create_current_appliance()
+    adb = appliance.db
+    vms = adb['vms']
+    hosts = adb['hosts']
+    storages = adb['storages']
+
+    path = ["Operations", "Virtual Machines", "Online VMs (Powered On)"]
+    report = CannedSavedReport.new(path)
+
+    vms_in_db = adb.session.query(
+        vms.name.label('vm_name'),
+        vms.location.label('vm_location'),
+        vms.last_scan_on.label('vm_last_scan'),
+        storages.name.label('storages_name'),
+        hosts.name.label('hosts_name')).join(
+            hosts, vms.host_id == hosts.id).join(
+                storages, vms.storage_id == storages.id).filter(
+                    vms.power_state == 'on').order_by(vms.name).all()
+
+    if len(vms_in_db) == len(list(report.data.rows)):
+        for vm in vms_in_db:
+            store_path = '{}/{}'.format(vm.storages_name.encode('utf8'),
+                                        vm.vm_location.encode('utf8'))
+            for item in report.data.rows:
+                if vm.vm_name.encode('utf8') == item['VM Name']:
+                    if (vm.hosts_name.encode('utf8') == item['Host'] and
+                        vm.storages_name.encode('utf8') == item['Datastore'] and
+                        store_path == item['Datastore Path'] and
+                        (str(vm.vm_last_scan).encode('utf8') == item['Last Analysis Time'] or
+                            (str(vm.vm_last_scan).encode('utf8') == 'None' and
+                             item['Last Analysis Time'] == '')
+                         )):
+                            continue
+                    else:
+                        pytest.fail("Found not matching items. db:{} report:{}".format(vm, item))
+    else:
+        pytest.fail("Lenghts of report and BD do not match. db count:{} report count:{}".format(
+            len(vms_in_db), len(list(report.data.rows))))
+
+
+@pytest.mark.tier(3)
+@test_requirements.report
+def test_datastores_summary(soft_assert, setup_a_provider):
+
+    appliance = get_or_create_current_appliance()
+    adb = appliance.db
+    storages = adb['storages']
+    vms = adb['vms']
+    host_storages = adb['host_storages']
+
+    path = ["Configuration Management", "Storage", "Datastores Summary"]
+    report = CannedSavedReport.new(path)
+
+    storages_in_db = adb.session.query(storages.store_type, storages.free_space,
+                                       storages.total_space, storages.name, storages.id).all()
+
+    if len(storages_in_db) == len(list(report.data.rows)):
```

Comment:
**Required**
Same pattern as the previous test, pytest.fail if the lengths don't match